packages/devtools/lib/src/debugger/debugger_state.dart

```diff
@@ -292,3 +290,34 @@ class DebuggerState {
     _breakpoints.add(isolate.breakpoints);
   }
 }
+
+class _Property<T> implements StreamSink<T> {
```

Comment:
This is an example implementation that would be suitable for sharing. I think we only use `add` so we could skip the `implements StreamSink`, I'll send that version as well. I'm fine with either doing something cheap locally or working on getting some shareable `Property` implemented somewhere.